internal/utils/src/wiremock_utils.rs

```diff
@@ -0,0 +1,186 @@
+use jsonrpsee::server::ServerHandle;
+use kakarot_rpc::run_server;
+use kakarot_rpc_core::lightclient::{
+    constants::{selectors::GET_STARKNET_CONTRACT_ADDRESS, ACCOUNT_REGISTRY_ADDRESS},
```

Comment:
My bad, in the latest update (today) ACCOUNT_REGISTRY_ADDRESS was deprecated, we don't use it anymore! 

No worries, depending on who merges first, this won't matter too much I can update it later on